src/chrome/content/rules/Surge.sh.xml

```diff
@@ -1,8 +1,7 @@
 <ruleset name="Surge.sh">
 
 	<target host="surge.sh" />
-	<target host="laughable-jam.surge.sh" />
-	<target host="www.surge.sh" />
+	<target host="*.surge.sh" />
```

Comment:
wildcard rule must have at least 3 `<test url>`. See https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/TheFreeDictionary.com.xml . My [understanding](http://surge.sh/help/using-https-by-default) is that each user has its own subdomain. would be nice to have a comment on top to explain the necessity of wildcard rule.

Reply:
you can use [Sublist3r](https://github.com/aboul3la/Sublist3r) to query the subdomains.